src/main/java/com/ibm/as400/access/QueuedMessageEnumeration.java

```diff
@@ -17,7 +17,7 @@
 import java.util.NoSuchElementException;
 
 // Helper class.  Used to wrap the QueuedMessage[] with an Enumeration.  This class is used by MessageQueue and JobLog.
-class QueuedMessageEnumeration implements Enumeration
+class QueuedMessageEnumeration implements Enumeration<Message>
```

Comment:
The type generic type should be `QueuedMessage` (not `Message`). This is the cause of the CI build failures as well

Reply:
Sure i will do the necessary changes